csdating.py

```diff
@@ -19,3 +19,26 @@ async  def sendhelp(self,message):
         msg = msg + "\n-->: Damit werden 3 Nachrichten mit den Zeiten 19 Uhr,20 Uhr und 21 Uhr schon mit Reactions gesendet!"
         await message.channel.send(msg)
         
+
+    async def reaction(self,payload):
+        #Get the Full message data 
+        message = await self.guilds[0].channels[2].fetch_message(payload.message_id)
```

Comment:
I got the same procedure of finding a messsage within rolehandler, maybe one (you?) could rather search for the channel, instead of hardcoding it. The channel order changes (iirc) whenenver you create a new channel or delete old ones.